CHANGES.txt

```diff
@@ -1,7 +1,12 @@
 Apache Falcon Change log
-For changes in releases after 0.9, please refer to "release-docs" or
+
+Future Release Version: 0.11
```

Comment:
Can we remove this line? Are we sure it will be 0.11 or 1.0?

Fix and ship it!